definitions/fabrikate-velero/README.md

```diff
@@ -0,0 +1,55 @@
+# Velero
+
+This [Fabrikate](https://github.com/microsoft/fabrikate) definition generates the Kubernetes resource manifests for [Velero](https://github.com/heptio/velero).
+
+## Usage
+
+Add the following to your component.yaml:
+
+```yaml
+subcomponents:
+  - name: "velero"
+    source: "https://github.com/sayar/fabrikate-velero"
```

Comment:
Let's adjust this to be how you would use this component when it is in this repo, ala:

```
  - name: velero
    source: https://github.com/microsoft/fabrikate-definitions
    path: definitions/fabrikate-velero
    method: git
```